services/server/server.go

```diff
@@ -47,6 +47,7 @@ import (
 	"gopkg.in/reform.v1"
 
 	"github.com/percona/pmm-managed/models"
+	"github.com/percona/pmm-managed/services/management/ia"
```

Comment:
We also try to not import one package from another to avoid risks of circle dependencies. Instead of that we use interfaces and mocks. Please check other services in the same file.

Reply:
I was forgotten there. For some reason it was not removed automatically on save.

Reply:
it's imported because you use it https://github.com/percona/pmm-managed/pull/626/files#diff-a990b29ab7e542cbefea97ccf6055f35a8f1486f1af794294dc542bf9e0e0b25R69